ospfd/ospfd.h

```diff
@@ -84,6 +84,9 @@ struct ospf_external {
 	struct route_table *external_info;
 };
 
+/* Status of various sub function e.g. TE */
+typedef enum _status_t {disable, enable} status_t;
```

Comment:
Just submit an update for this PR. Hope it will cover all requested modifications.